content/04.network_applications.md

```diff
@@ -10,7 +10,31 @@
 
 #### Matrix Factorization
 
-1. Mention techniques for these with some papers
+Matrix factorization is a technique that uses linear algebra to map high dimensional data into a low dimensional space.
+This projection is accomplished by decomposing a matrix into a set of small rectangular matrices.
+In the context of knowledge graphs the input matrix would be an adjacency matrix where the rows and columns are nodes and each entry in this matrix represents the presence of an edge between two nodes.
+The output of this technique would be a small matrix where each row represents a low dimensional embedding for every node in the adjacency matrix.
```

Comment:
Hmm - I usually think of this approach as decomposing a large matrix into two smaller matrices. Do you mean something different than this?